src/components/v2v3/V2V3Project/V2V3ManageTokensSection/index.tsx

```diff
@@ -320,9 +323,12 @@ export default function V2ManageTokensSection() {
         redeemDisabled={redeemDisabled}
         tokenSymbol={tokenSymbol}
         tokenAddress={tokenAddress}
+        tokenUnclaimedBalance={unclaimedBalance}
+        transferUnclaimedTokensTx={useTransferUnclaimedTokensTx}
         RedeemModal={V2RedeemModal}
         ClaimTokensModal={V2ClaimTokensModal}
         MintModal={V2MintModal}
+        TransferUnclaimedTokensModal={TransferUnclaimedTokensModal}
```

Comment:
**Issue:** I don't think this prop is necessary. Since this component is shared it can just be imported within the shared `ManageTokensModal`

Reply:
Oooh, you are right, my bad!